xorriso/gen_input.sh

```diff
@@ -34,11 +34,13 @@ then
     popd > /dev/null
 fi
 
-if [ -f $(pwd)/sb_pubkey.der ]
+pushd "${PWD}/../" > /dev/null
```

Comment:
Changing directory makes sense when `find` is used to make the loop easier. You're not looping over anything. You just add the `..` to the `if` statement and `map` command

Reply:
Sounds good, I can make that change.